src/process/plumerise/CCPr_Plumerise_Mod.F90

```diff
@@ -0,0 +1,271 @@
+!> \file CCPr_Plumerise_mod.F90
+!! \brief Driver for the CATCHem Process: Plumerise
+!!
+!! \defgroup catchem_plumerise_process
+!!
+!! The CATChem Plumerise Process group holds all the CATCHem Plumerise processes.
+!!!>
+MODULE CCPr_Plumerise_mod
+
+   ! USES:
+   USE State_Mod
+   USE Error_MOD
+
+   IMPLICIT NONE
+   PRIVATE
+
+   PUBLIC :: CCPR_Plumerise_Run
+   PUBLIC :: CCPR_Plumerise_Init
+   PUBLIC :: CCPR_Plumerise_Finalize
+
+   TYPE, PUBLIC :: PlumeRiseStateType
+      Logical :: activate      !< Activate Plume Rise
+   END TYPE PlumeRiseStateType
+
+CONTAINS
+
+   !>
+   !! \brief Initialize the CATCHem Plumerise Process
+   !!
+   !! \param Config_Opt  CATCHem configuration options
+   !! \param PlumeriseState   CATCHem Plumerise state
+   !! \param ChemState    CATCHem chemical state
+   !! \param RC          Error return code
+   !!
+   !! \ingroup catchem_plumerise_process
+   !!!>
+   SUBROUTINE CCPR_Plumerise_Init(PlumeriseState, RC)
+      ! USES
+
+      IMPLICIT NONE
+
+      ! INPUT PARAMETERS
+      !-----------------
+      TYPE(PlumeriseStateType), intent(inout) :: PlumeriseState  !< Nullify Plumerise State During INIT
+
+      ! INPUT/OUTPUT PARAMETERS
+      !------------------------
+      INTEGER,          INTENT(INOUT) :: RC
+
+      ! LOCAL VARIABLES
+      !----------------
+      INTEGER :: k ! Loop Counter
+
+      ! Error handling
+      !---------------
+      CHARACTER(LEN=255)    :: ErrMsg
+      CHARACTER(LEN=255)    :: ThisLoc
+
+      ! Initialize Error handling
+      !--------------------------
+      ErrMsg = ''
+      ThisLoc = ' -> at CCPR_Plumerise_INIT (in process/Plumerise/ccpr_Plumerise_mod.F90)'
+
+      ! Initialize
+      !-----------
+      if (Config%Plumerise_activate) then
+
+         ! Activate Plumerise Process
+         !----------------------
+         Plumerisestate%Activate = .true.
```

Comment:
would "active" or "activated" be a better word for such property?

Reply:
this is the same nomenclature that we have used in other processes.  suggest we stick to it